vulture_os/system/config/api.py

```diff
@@ -62,7 +62,7 @@ def get(self, request):
             }, status=500)
 
     @api_need_key('cluster_api_key')
-    def put(self, request, object_id):
+    def put(self, request, object_id=None):
```

Comment:
method needs to apply to an existing object

```suggestion
    def put(self, request, object_id):
```

Reply:
object_id is useless here because there is always only one config on the cluster.
It may be great to delete it.

Reply:
I deleted this argument, do you agree ?